### DISCUSSION TITLE
How to suppress api call output?

### DISCUSSION THREAD

**Discussion by dghjunior** [theOehrly/Fast-F1#176] [Q&A]
Hello, I have just started working with this package and I absolutely love it, but I was curious if there is a way to hide api call outputs. For example the lines like this:
"api                INFO      No cached data found for car_data. Loading data..."

Any suggestions or should I just go for workarounds?

**Comment by theOehrly** [comment]
Yes, there is. FastF1 uses pythons built-in 'logging' module.
You can do the following:

```
import logging 
import fastf1
[... other imports]

logging.disable(logging.INFO)

[... your code]
```

This will disable the INFO outputs, but will still print WARNINGs (I highly recommend that).
